notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>Use the following way to set a default TensorBoard instance otherwise the instance won't be able to set correctly in cloud build tests.</p><p> </p><pre class="ql-syntax" spellcheck="false">TENSORBOARD_NAME = "[your-tensorboard-name]"  # @param {type:"string"}
</pre><p>In&nbsp;[&nbsp;]:</p><pre class="ql-syntax" spellcheck="false">if (
    TENSORBOARD_NAME == ""
    or TENSORBOARD_NAME is None
    or TENSORBOARD_NAME == "[your-tensorboard-name]"
):
    TENSORBOARD_NAME = PROJECT_ID + "-tb-" + TIMESTAMP
</pre><p>If you don't have a TensorBoard instance, create one by running the following cell:</p><p>In&nbsp;[&nbsp;]:</p><pre class="ql-syntax" spellcheck="false">tensorboard = aiplatform.Tensorboard.create(
    display_name=TENSORBOARD_NAME, project=PROJECT_ID, location=REGION
)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>done</p>